speedtest.py

```diff
@@ -341,10 +341,6 @@ class SpeedtestBestServerFailure(SpeedtestException):
     """Unable to determine best server"""
 
 
-class SpeedtestMissingBestServer(SpeedtestException):
-    """get_best_server not called or not able to determine best server"""
```

Comment:
I'm ok accepting this change, however I think we temporarily need to keep this exception.

If someone was importing this exception, and we remove it, that would cause issues.

We'd need a note that it should be removed in the next major release.

Reply:
Sounds good. I'll add it back now.